### PR TITLE
dev-pt-2379 container orchestration

### LINKED ISSUE #2379: [pt] Localize container orchestration
Localize https://glossary.cncf.io/container-orchestration/.

### PR DESCRIPTION
**Pull request by fharris**
### Describe your changes
Localize https://glossary.cncf.io/container-orchestration/.

### Related issue number or link (ex: `resolves #issue-number`)
resolves #2379 

### Checklist before opening this PR (put `x` in the checkboxes)
- [X] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [X] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/pt-br/container-orchestration.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: Orquestração de Contêiners
+status: Completed
+category: conceito
+---
+
+
+Orquestração de [Contêiners](/container/) refere-se à gestão e automatização do ciclo de vida de aplicativos [contêinerizados](/containerization/) em ambientes dinâmicos. 
+É executado através de um Orquestrador de Contêiners ([Kubernetes](/kubernetes) na maioria dos casos). Plataformas que possibilitam e facilitam a implantação controlada de multiplos aplicativos,  o seu [escalonamento](/scalability/) automatizado, a recuperação automática em caso de falha e monitoramento.
```
Comment (edsoncelio) [inline content/pt-br/container-orchestration.md:9]: The links need to be updated to point to the pt-br version, like:
* pt-br/container
* pt-br/containerization
* pt-br/kubernetes
* pt-br/scalability

Please take a look and update the other occurrences.

File: content/pt-br/container-orchestration.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+A utilização do termo Orquestração serve neste contexto como metáfora para descrever o papel deste tipo de ferramenta na condução dos varios elementos, normalmente de natureza distribuída, como se de um verdadeiro maestro  se tratasse, onde cada contêiner se comporta como um músico da orquestra. 
+
+## Problema relacionado
+
+A gestão em escala de [microserviços](/microservices) e em geral de qualquer outro [sistema distribuído](/distributed-systems), incluindo segurança e comunicação de rede, é algo muito difícil, senão impossível, de se fazer manualmente. As plataformas de orquestração de contêiners possibilitam a automatização de este tipo de tarefa de administração.
+
+## Como isso ajuda
+
+Plataformas e respectivas ferramentas de orquestração de contêiners, são eminentemente declarativas. Isto é, permitem declarar o estado desejável do sistema, como por exemplo o número de containers, a quantidade de réplicas de um determinado pod, etc.
```
Comment (edsoncelio) [inline content/pt-br/container-orchestration.md:18]: "containers" -> "contêineres"

**Review by edsoncelio — CHANGES_REQUESTED** [review]
Hi @fharris 
I made a few comments before approving it.
Please take a look and in case of doubt let me know.

**Comment by fharris** [comment]
Thank you @edsoncelio . Changed accordingly to your review.

**Review by edsoncelio — APPROVED** [review]
Thanks!

**Pull request merged**